### ISSUE TITLE
Not working on High Sierra

### ISSUE THREAD

**Issue by moisesrodriguez** [bwesterb/blame-bird#7]
Been using this script for a while. But once I upgraded to High Sierra I get this error

```
Traceback (most recent call last):
  File "blame-bird.py", line 85, in <module>
    sys.exit(main())
  File "blame-bird.py", line 43, in main
    'select item_id, zone_rowid from client_unapplied_table'):
sqlite3.OperationalError: no such column: item_id
```

**Comment by bwesterb** [comment]
They changed the structure of the index.  My bird folder is quite empty all the time, so it's hard for me to reverse engineer the new index format.

**Comment by jendas1** [comment]
Please update the README that this code doesn't work on High Sierra.

**Comment by intari** [comment]
Same issue on Mojave

**Comment by bwesterb** [comment]
Might be fixed by #10

**Comment by royhuang1** [comment]
Note that the latest script doesn't work on OS X Sierra.
